### ISSUE TITLE
Windows Desktop app not running as expected...

### ISSUE THREAD

**Issue by Lewiscowles1986** [turtl/desktop#7]
Ran the following commands after extracting files for `js`, `mobile` and `desktop` to a central turtl folder i.e. `turtl\desktop` `turtl\mobile` `turtl\js`)

created symbolic links to folders using mklink

```
cd mobile
MKLINK /D .\www\app ..\..\js
cd ..\..\desktop
MKLINK /D .\data\app ..\js
```

downloaded nwjs binary from github as npm did not seem to install... Installed to `turtl\nwjs-64`

```
..\nwjs-64\nw.exe .
```

![image](https://cloud.githubusercontent.com/assets/2605791/8982950/e20d3564-36bd-11e5-9be7-e00e3f225236.png)

**Comment by orthecreedence** [comment]
You may have to run `./scripts/generate-templates` after creating your symlinks (sorry, the desktop docs are outdated, and most people haven't bothered to try running from source).

`Ctrl+shift+k` will open the JS console and let you see what's going on, but my guess is missing templates and/or some kind of path issue.

**Comment by Lewiscowles1986** [comment]
Right, I will try to combine that with grunt tasks for mobile as well ;)

**Comment by Lewiscowles1986** [comment]
Also I think it is that way because of LISP backend...
